js/views/userPageVw.js

```diff
@@ -505,7 +504,7 @@ module.exports = Backbone.View.extend({
 
     $.ajax({
       type: "DELETE",
-      url: self.item.get('server') + "contracts/?id="+ self.item.get('id'),
+      url: self.item.get('server') + "contracts/?id="+ self.item.get('id'), //TODO: Change from GET to POST for safe methods, http://restcookbook.com/HTTP%20Methods/idempotency/
```

Comment:
DELETE, as used here, is fine. If you want to modify the API to accept a POST with a parameter that would delete the contract instead, you can put in a pull request that does that.